sdk/javascript/src/impl/CipherHelpers.js

```diff
@@ -0,0 +1,68 @@
+const { AesGcmCipher, AesCbcCipher } = require('../Cipher');
+const crypto = require('crypto');
+
+const GCM_IV_SIZE = 12;
+const CBC_IV_SIZE = 16;
+const SALT_SIZE = 32;
+
+const concatArrays = (...arrays) => {
```

Comment:
thrown it here, but maybe we want it somewhere else

Reply:
in this case it probably should be somewhere else, cause CipherHelpers should not return concatenated things (and concat is needded in MessageEncoders)